participants/Fouesneau/pr_tutorial/simple_functions.py

```diff
@@ -1,11 +1,17 @@
-def fibonacci(max):
+from typing import Sequence
+
+def fibonacci(max: int) -> List[int]:
```

Comment:
I think `List` is missing its import and `Sequence` is unused. I'd suggest
```suggestion
from typing import List

def fibonacci(max: int) -> List[int]:
```